charm/kafka/layer.yaml

```diff
@@ -15,6 +15,7 @@ options:
   snap:
     core: {}
     kafka:
+      channel: beta
```

Comment:
@cmars Why are we using the beta channel? Shouldn't we just upgrade stable?

Reply:
I've decided to release the snaps to the same channels as our charms, to keep releases simple. So edge charm gets edge snap as a resource, likewise for beta. I feel like our snaps are beta quality (pretty good, but not yet battle tested).

I've closed the stable channel for kafka, zk, etc. It's all edge or beta now.

Reply:
Main reason I'm setting beta as the default here is in case someone does a charm deploy without specifying a resource, and the layer falls back on the snap store, we default to a safe known good release.